### DISCUSSION TITLE
Changes to header and footer in the TSML UI

### DISCUSSION THREAD

**Discussion by OSAwebsite** [code4recovery/12-step-meeting-list#1060] [General]
Hi Team,

I have taken some time to inspect what is different in the TSML UI, the new interface we can toggle on or off.
I like how it looks and it seems a lot faster as well than the old setup.
However, one of the changes I have noticed is something I would rather see differently.
It is the (apparently removed) option to create text in the header and the footer of the meeting list, and in the footer of the details page. Again: this is in the new design that I am currently not yet using on my live website.

What is the reason to make the information in header and footer the same for the list of meetings and the details page of 1 meeting? Both the list and the details page serve a different purpose, or so it feels to me.

In the list, I show a number of clickable links in the footer that users can use to quickly find all in-person meetings; or hybrid; or online. Once they are in the details page of a single meeting, those links serve no purpose. I use the bottom widget in the details page to remind members to use the online Change Notification form we have for record changes.

I would rather have the option to have different headers and footers for the list versus the details page.

Is that possible?
Nicky

**Comment by joshreisner** [comment]
Yep, tracking this issue here: https://github.com/code4recovery/12-step-meeting-list/issues/686
